scripts/test

```diff
@@ -7,7 +7,7 @@ echo Running tests
 
 # s390x doesn't support -race flag
 if [ "${ARCH}" != "s390x" ]; then 
-    go test -race -cover -tags=test ./...
```

Comment:
Is there a reason why don't want to test for race conditions while running the tests?

Reply:
And in case we do not want to use `-race`, the condition can be removed and simply replaced with `go test -cover -tags=test ./...`.

Reply:
It was failing miserably for `s389x` when testing race conditions.

> And in case we do not want to use `-race`, the condition can be removed and simply replaced with `go test -cover -tags=test ./...`.

yup, that's what I did

Reply:
I've removed the `if`...`else` because both were doing the same